<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPaintingBoardClass</class>
 <widget class="QMainWindow" name="MyPaintingBoardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyPaintingBoard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <item>
     <widget class="MyGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,0,1,1,1,6">
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="colorLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bucketColor">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>colorLabel</zorder>
         <zorder>colorLabel</zorder>
         <zorder>bucketColor</zorder>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,2">
          <item>
           <widget class="QPushButton" name="colorSetButton">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>设置边框颜色</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backgroundColorButton">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>设置填充颜色</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bucketColorSet">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>设置油漆桶颜色</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>线条宽度:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineWidth">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>长度:   </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>宽度：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="pageSetting"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="createTool"/>
   <addaction name="openTool"/>
   <addaction name="saveTool"/>
   <addaction name="moveTool"/>
   <addaction name="selectTool"/>
   <addaction name="lineTool"/>
   <addaction name="rectTool"/>
   <addaction name="ellipseTool"/>
   <addaction name="triangleTool"/>
   <addaction name="textTool"/>
   <addaction name="paddingTool"/>
   <addaction name="groupTool"/>
   <addaction name="unGroupTool"/>
   <addaction name="magnifyTool"/>
   <addaction name="shrinkTool"/>
   <addaction name="commitTool"/>
   <addaction name="rollbackTool"/>
   <addaction name="copyItems"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开项目</string>
   </property>
   <property name="toolTip">
    <string>操作以打开现有项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>操作以保存当前项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="pageSetting">
   <property name="text">
    <string>页面设置</string>
   </property>
   <property name="toolTip">
    <string>操作以设置页面</string>
   </property>
  </action>
  <action name="createTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/新建颜色_图层.png</normaloff>:/MyPaintingBoard/resource/新建颜色_图层.png</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="saveTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/保存.png</normaloff>:/MyPaintingBoard/resource/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="openTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/打开文件夹.png</normaloff>:/MyPaintingBoard/resource/打开文件夹.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="lineTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/图形-直线.png</normaloff>:/MyPaintingBoard/resource/图形-直线.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="rectTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/图形-矩形.png</normaloff>:/MyPaintingBoard/resource/图形-矩形.png</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="ellipseTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/图形-圆.png</normaloff>:/MyPaintingBoard/resource/图形-圆.png</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="triangleTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/形状-三角形.png</normaloff>:/MyPaintingBoard/resource/形状-三角形.png</iconset>
   </property>
   <property name="text">
    <string>三角形</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="textTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/文字.png</normaloff>:/MyPaintingBoard/resource/文字.png</iconset>
   </property>
   <property name="text">
    <string>文字</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="paddingTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/油漆桶.png</normaloff>:/MyPaintingBoard/resource/油漆桶.png</iconset>
   </property>
   <property name="text">
    <string>油漆桶</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="moveTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/移动.png</normaloff>:/MyPaintingBoard/resource/移动.png</iconset>
   </property>
   <property name="text">
    <string>移动</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="selectTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/选区.png</normaloff>:/MyPaintingBoard/resource/选区.png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="magnifyTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/放大.png</normaloff>:/MyPaintingBoard/resource/放大.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="shrinkTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/缩小.png</normaloff>:/MyPaintingBoard/resource/缩小.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="commitTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/后退.png</normaloff>:/MyPaintingBoard/resource/后退.png</iconset>
   </property>
   <property name="text">
    <string>前进</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="rollbackTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/前进.png</normaloff>:/MyPaintingBoard/resource/前进.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="testRect">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="groupTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/组合.png</normaloff>:/MyPaintingBoard/resource/组合.png</iconset>
   </property>
   <property name="text">
    <string>组合</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="unGroupTool">
   <property name="icon">
    <iconset resource="MyPaintingBoard.qrc">
     <normaloff>:/MyPaintingBoard/resource/GraphicsGroup.h</normaloff>:/MyPaintingBoard/resource/GraphicsGroup.h</iconset>
   </property>
   <property name="text">
    <string>打散</string>
   </property>
  </action>
  <action name="copyItems">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteItems">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyPaintingBoard.qrc"/>
 </resources>
 <connections/>
</ui>
